doc/data/messages/u/unsupported-membership-test/good.py

```diff
@@ -1 +1,8 @@
-# This is a placeholder for correct code for this message.
+class Fruit:
+    FRUITS = ["apple", "orange"]
+
+    def __getitem__(self, name):
+        return name in self.FRUITS
+
+
+apple = "apple" in Fruit()
```

Comment:
When I execute this file, the python interpreter just hangs.

Reply:
huh! you're right! even hangs in the REPL. I'll investigate but will change the example

Reply:
looks like an infinite loop 🤦‍♀️

Reply:
Btw, the primer should finally be working now if you merge main. Sorry for bringing it down earlier this week!

Reply:
oh thank you!! that's excellent news as there's lots of PRs waiting for it